### PR TITLE
Fixed documentation compliance

### LINKED ISSUE #66: Fix documentation compliance
Per https://github.com/BHoM/Excel_Toolkit/pull/60/checks?check_run_id=23124193055

### PR DESCRIPTION
**Pull request by Tom-Kingstone**
### Issues addressed by this PR

Closes #66 

fixed input/output attributes being out of order for some methods. Also added attributes to ListToText.

### CODE REVIEW

File: Excel_Engine/Compute/ListToText.cs [hunk 1]
```diff
@@ -38,6 +38,10 @@ public static partial class Compute
         /**** Public Methods                    ****/
         /*******************************************/
 
+        [Description("Converts a list of objects into a single string using the delimiter provided to separate items.")]
+        [Input("objects", "The list of objects to be converted to a string.")]
+        [Input("delimiter", "The delimiter to be used that separates the items of the list.")]
+        [Output("text", "The output text.")]
         public static string ListToText(List<object> objects, string delimiter=",")
```
Comment (FraserGreenroyd) [inline Excel_Engine/Compute/ListToText.cs:45]: Can we add some space around `= ","` - before and after the `=` ideally.

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]

**Comment by Tom-Kingstone** [comment]
@BHoMBot check core
@BHoMBot check compliance

**Review by FraserGreenroyd — APPROVED** [review]
All looks good now, thanks @Tom-Kingstone - approving based on code review.

**Comment by FraserGreenroyd** [comment]
@BHoMBot check required

**Comment by FraserGreenroyd** [comment]
@BHoMBot check ready-to-merge

**Pull request merged**